blog/authors.yml

```diff
@@ -40,6 +40,7 @@ Maxr1998:
 
 nielsvanvelzen:
   name: Niels van Velzen
+  title: Android lead
```

Comment:
We can have titles!? Sign me up! 😆

Reply:
Yeah I didn't know about it until recently. I've updated it to capitalize Lead. If you want a title of your own you should open a separate PR 😉